composer.json

```diff
@@ -34,7 +35,7 @@
         "sonata-project/user-bundle": "For user timezone detection"
     },
     "conflict": {
-        "sonata-project/user-bundle": "<2.0 || >=4.0"
+        "sonata-project/user-bundle": "<2.0 || >=5.0"
```

Comment:
Can we get your opinion on this @greg0ire @OskarStark 

We shouldn't rely on one person only for general decisions, because @Soullivaneuh is very busy :(

For me no upper limit is no big deal